WordPressMocks/src/main/assets/mocks/mappings/wpcom/sites/rest_v2_sites_106707880_dashboard_cards.json

```diff
@@ -0,0 +1,16 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPattern": "/wpcom/v2/sites/(.*)/dashboard/cards/\\?_locale=(.*)"
```

Comment:
Note that we are using `urlPattern` matcher here, instead of usually used `urlPath` or `urlPathPattern` because we want to create a matcher where `_locale` is present right after the path.

If we had something like this instead:

```
  "request": {
    "method": "GET",
    "urlPathPattern": "/wpcom/v2/sites/(.*)/dashboard/cards/",
    "queryParameters": {
      "_locale": {
        "matches": "(.*)"
      }
    }
  }
```

This could potentially match a set of other requests, where `_locale` is present at the end of the query, preceded by other parameters.

Reply:
🤔  I'm wondering, why is this strictly necessary to have that parameter right after the path though?

Given how the URL standard works, the order of the query parameters should not be semantically significant, and could appear in any order without changing the canonical form and meaning of the URL, so why enforcing this one to be right after the path?

Reply:
In this particular case, we are adding a mock file to match a request that has `_locale` right after the url, and it is the _only_ parameter. At least in Woo, I saw no mocks that contained the only parameter being locale. So, this is a unique case.

On the other hand, we do have lots of other requests where there will be a lot of parameters, but `locale` will always be in the end. While you correctly mention that their order is not strict, in practice the `locale` was always in the end. So, I've decided to rely on that, and use this point in the matcher. I'll not check if there's a WireMock feature allowing to specify somehow that we need to have only one parameter.

Reply:
*
> I'll not check
I'll now* check

🤦

Reply:
Ah right ok, makes sense then. If there's a way to specify to WireMock that there should be only one parameters, or to make sure only the parameters you listed are present and none other, then I'll go for it… otherwise I guess `urlPattern` is good enough (and exists for those kind of cases I guess) 🙂